static/lib/jquery.autocomplete.js

```diff
@@ -12,34 +12,33 @@
 // Expose plugin as an AMD module if AMD loader is present:
```

Comment:
Is there a new version available of the autocomplete?

Reply:
@Tatider We have the [v1.4.1](https://github.com/devbridge/jQuery-Autocomplete/releases/tag/v1.4.1) release on 3 Apr 2017 for Ajax Autocomplete for jQuery Plugin, his last release was [v1.4.11](https://github.com/devbridge/jQuery-Autocomplete/releases/tag/v1.4.11) on 1 May 2020